src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -1346,13 +1346,110 @@ ORDER BY shardid, nodeport;
 \c - - - :master_port
 INSERT INTO pg_dist_shard_placement (SELECT * FROM tmp_shard_placement);
 DROP TABLE tmp_shard_placement;
+-- Show that MX tables can be created with the old distributed table creation APIs
+-- First show that we can create a streaming distributed table
+SET citus.replication_model TO 'streaming';
+SET citus.shard_replication_factor TO 1; -- Prevent the ingenuine error
+CREATE table mx_old_api(a INT, b BIGSERIAL);
+SELECT master_create_distributed_table('mx_old_api', 'a', 'hash');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT repmodel FROM pg_dist_partition WHERE logicalrelid='mx_old_api'::regclass;
+ repmodel 
+----------
+ s
+(1 row)
+
+-- Show that the metadata is created on the worker 
+\c - - - :worker_1_port
+\d mx_old_api
+                         Table "public.mx_old_api"
+ Column |  Type   |                       Modifiers                        
+--------+---------+--------------------------------------------------------
+ a      | integer | 
+ b      | bigint  | not null default nextval('mx_old_api_b_seq'::regclass)
+
+SELECT repmodel FROM pg_dist_partition WHERE logicalrelid='mx_old_api'::regclass;
+ repmodel 
+----------
+ s
+(1 row)
+
+-- Create the worker shards and show that the metadata is also on the workers
+\c - - - :master_port
+SELECT master_create_worker_shards('mx_old_api', 4, 1);
```

Comment:
Correct, with #1201 we can overcome it but it might end up causing more problems if combined with other commands so I think it requires more careful testing. Are you OK if we skip solving this case for now and document it in an issue?